clients/web/src/components/programmatic-ad/mozads-ad-slot.tsx

```diff
@@ -0,0 +1,67 @@
+import { css } from '@emotion/css'
+import { useTranslation } from 'next-i18next'
+
+import { MozAdsPlacement } from '@mozilla-services/majc/dist/react'
+import { IABFixedSize, MozAdsSize } from '@mozilla-services/majc/dist/core'
+
+const programmaticAdWrapperStyles = css`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  & > .label {
+    font-family: var(--fontSansSerif);
+    font-size: 0.85rem;
+    line-height: 100%;
+    color: var(--color-textTertiary);
+    text-align: center;
+    margin: 0;
+    padding: 0.5rem 0;
+    width: 100%;
+  }
+`
+
+// Syndicated Article MozAds placements
+export enum Placements {
+  ABOVE_THE_FOLD = 'mock_pocket_billboard_1',
```

Comment:
Reminder comment to put the real placement names here before merging this PR.

Reply:
@mashalifshin do we have real placement names yet?